src/components/Task/index.jsx

```diff
@@ -1,23 +1,45 @@
+import { TaskInfo } from './TaskInfo';
 import React from 'react';
 import PropTypes from 'prop-types';
 
 // Styled
-import { TaskSC } from './style';
+import { TaskSC, TitleSC, TagsSC, StatusSC } from './style';
+// Components
+import Tags from '../DetailsViewBody/Tags';
+import Status from '../Status';
 
 
 
-const Task = ({ changeOpenedTaskId, id, title }) => {
+const Task = ({
+    changeOpenedTaskId,
+    openedTaskId,
+    task: { id, tags, todo, title, assignees, status, created }
+}) => {
     return (
-        <TaskSC data-id={id} onClick={() => changeOpenedTaskId(id)}>
-            {title}
+        <TaskSC
+            data-id={id}
+            isOpened={openedTaskId === id}
```

Comment:
move logic to the parent component, move `isOpened` flag to the props instead of the `openedTaskId`